kiwi/schema/kiwi.rnc

```diff
@@ -1002,12 +1002,12 @@ div {
         }
     k.repository.alias.attribute =
         ## Alias name to be used for this repository. This is an
-        ## optional free form text. If not set the source attribute
+        ## optional POSIX form text. If not set the source attribute
```

Comment:
This is still free-form text, it's just restricted on what characters are allowed in the field.